index.html

```diff
@@ -14,46 +14,190 @@
       rel="stylesheet"
     />
 
-    <title>portfolio-setup Project</title>
+    <title>portfolio-setup</title>
   </head>
   <body>
-    <!-- my-logo section -->
-    <header class="toolbar-section">
-      <div class="logo">
-        <a href="">My Logo</a>
-      </div>
-
-      <div>
-        <p><span class="iconify" data-icon="eva:menu-outline"></span></p>
-      </div>
-    </header>
-
-    <!-- main section -->
-    <main class="main-section">
-      <div class="content">
-        <p class="author">
-          I’m Ismail <br />
-          Glad to see <br />
-          you!
+    <!-------------------------------------- middle-section ---------------------------------------->
+
+    <section class="middle-section">
+      <!-- tonic section 1 -->
+      <section class="tonic-section">
+        <div class="tonic-image"></div>
+        <h2 class="tonic">Tonic</h2>
+        <div class="canopy-container">
+          <p>CANOPY</p>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <ul class="button-container">
+          <li><button class="html">html</button></li>
+          <li><button class="css">css</button></li>
+          <li><button class="javascript">javascript</button></li>
+        </ul>
```

Comment:
One of the major project requirements is that you make your portfolio match the Figama template that you choose. Currently, it doesn't match and I would kindly suggest that you follow the design thoroughly 🙏.

- [ ] The design uses `font-family` of `Poppins` and different `font-weights` and `font-sizes` for different parts, kindly import it and use it to build your portfolio.
- [ ] The design uses different `margins`, `paddings`, and `colors` for each element, kindly use the same measurements and colors for each of the elements in your portfolio.

## Your Design

![image](https://user-images.githubusercontent.com/77106962/143556239-69e8edef-beb9-45be-8dfa-f76cfd166aa6.png)

## Figma Template

![image](https://user-images.githubusercontent.com/77106962/143556507-a6eb1e51-7dc3-4f2d-a876-9e108da2c8d9.png)

### Please adjust your design to match it.